src/plugins/unified_search/public/typeahead/suggestion_component.test.tsx

```diff
@@ -8,7 +8,7 @@
 
 import { mount, shallow } from 'enzyme';
 import React from 'react';
-import { QuerySuggestion, QuerySuggestionTypes } from '../../../data/public';
+import { QuerySuggestion, QuerySuggestionTypes } from '../../../unified_search/public';
```

Comment:
Now these are located in the same plugin so this import can be simpliied

Reply:
Thank you for your comment. Updated.